.github/workflows/backend.yml

```diff
@@ -324,6 +324,32 @@ jobs:
         run: |
           make test-plugins
 
+  region-to-control:
```

Comment:
is this a required check? as written it is not and so a merge will succeed with this failing

can these tests be added to an existing job rather than making a new special one?

Reply:
Good question.  The main reason this one test needs to be separate from the other backend jobs it he requirement of kafka.  However, it does not cleanly fit into other sections (relay, etc) that are more specialized.  I'd love to consolidate, but it didn't seem to work.

I can add required to the check.

Reply:
hmmm interesting

I wonder if we can't just run kafka for everything -- it's kind of a pain to figure out the right set of things to run tests locally already.  adding one to the number isn't the worst but I wonder if there's a way to land this without doing that

Reply:
🤷  I mean, I don't disagree.  But conversely, I don't know, other than having a big conversation with dev-infra, how to create consensus to pass this through the eye of the needle.  It's in that grey area where you could argue multiple solutions to the same problem (enable kafka in all test environments, widen an existing test group, adding this test group) such that you might bike shed a lot without a real conclusion.

Considering that changing this after the fact is trivial, and there isn't substantive impact at this point one way or another, I'm really just looking for any definitive answer without too much bike sheding.

Reply:
The `snuba` test jobs already spin up kafka. These tests could be added to that build group if you didn't want to add another one./m

Reply:
Question then: should I also -rename- the test category -and- the make target?  Since it would become a broader category and adding this to that would be a misnomer.  In that case, it also means that existing users may need to "relearn" which make target to use to test snuba.

I guess, honestly, I'm skeptical that making a name or categorical change is attractive compared to adding the new category.  For instance, the category splitting by subject is attractive because changes will tend to break things along specific categories, and rerunning tests within a narrow category is attractive compared to larger ones.